packages/delegation-toolkit/test/DelegationFramework/DelegationManager/delegationManagement.test.ts

```diff
@@ -0,0 +1,152 @@
+import { isStrictHexString } from '@metamask/utils';
```

Comment:
I think I probably misled you here a little - @metamask/utils `isStrictHexString` is correctly used in @metamask/delegation-core, but in @metamask/delegation-toolkit we should probably use viem's `isHex` function - their docs appear to be down rn, https://viem.sh/docs/utilities/isHex.html but there's an optional `{ strict: bool }` parameter which defaults to true.